src/types.ts

```diff
@@ -1200,6 +1218,9 @@ export class ZodString extends ZodType<string, ZodStringDef, string> {
   get isBase64() {
     return !!this._def.checks.find((ch) => ch.kind === "base64");
   }
+  get isBase64url() {
```

Comment:
This (and `base64url()`) above could use a comment. If you haven't heard of `base64url` before, you might assume this is some sort of URL (`e.g. `data:base64,abc`) rather than a text encoding of binary data.

Reply:
Agreed, added in 05c4922fa36a225b5671d9b4c5d160d5ef563cd1.